cobra/test/test_flux_analysis.py

```diff
@@ -315,6 +376,25 @@ def test_single_gene_deletion_benchmark(self, model, benchmark,
         model.solver = solver
         benchmark(single_reaction_deletion, model=model, processes=1)
 
+    @pytest.mark.parametrize("solver", optlang_solvers)
+    def test_single_reaction_deletion_room(self, solver):
+        model = construct_papin_2003_model()
+        model.solver = solver
+        wt_sol = construct_papin_2003_solution()
```

Comment:
You don't actually use the `wt_sol` and thus also don't need the `construct_papin_2003_solution` function.

Reply:
The `wt_sol` is being used as the reference solution when performing `add_room`.

Reply:
I overlooked that, sorry.